### ISSUE TITLE
Document should contain landmarks

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#2427]
## Summary

With modern assistive technology, landmarks provide a simple way to navigate a webpage. The technologies allow users to navigate between HTML5 landmarks such as `<header>`, `<nav>`, `<main>` and `<footer>` landmarks.

Currently we have `<nav>`, `<header>` and `<footer>` but lack a `<main>` landmark.

This would work in conjunction with #2426 acting as a fallback for assistive technologies that do not support navigating by landmarks.

Related: #2426 

Ref: [Page must have one main landmark](https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome)

## Proposed Solution

This would easily be solved by wrapping our router `{content}` in a `<main>`

```jsx
<main>
  {content}
</main>
```
